subsystem/agent_based_installer_client_test.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("RegisterClusterAndInfraEnv", func() {
 		Expect(registerClusterErr).NotTo(HaveOccurred())
 		Expect(network.GetApiVipById(&common.Cluster{Cluster: *modelCluster}, 0)).To(Equal("1.2.3.8"))
 		Expect(network.GetIngressVipById(&common.Cluster{Cluster: *modelCluster}, 0)).To(Equal("1.2.3.9"))
-		Expect(modelCluster.OpenshiftVersion).To(ContainSubstring(snoVersion))
+		Expect(modelCluster.OpenshiftVersion).To(ContainSubstring("4.15.0"))
```

Comment:
Why was this changed?

Reply:
It needs to match the version inside `../docs/hive-integration/crds/clusterImageSet.yaml`, which I upgraded

Reply:
I'm not even sure that needs to change.